packages/common/src/utils/index.ts

```diff
@@ -1,14 +1,16 @@
 import { v4 } from 'uuid';
 import { isEqual } from 'lodash';
 
-export * from './NumUtils';
+export * from './numbers';
 export * from './quantities';
 export * from './formatters';
 export * from './testing';
-export * from './debounce';
-export * from './dateFunctions';
+export * from './dates';
 export * from './arrays';
 export * from './regex';
+export * from './pricing';
+export * from './functions';
+export * from './navigation';
 
 export type UUID = string;
```

Comment:
There's some extras in here that I wanted to organise but I ran out of ideas. Got any?

Reply:
do you mean the rest of the bits in the index?

- `UUID` could be in types
- `isTypeOf` could be in types as well. it is a util fn too though, so 🤷 
- `isEqual` in functions
- `generateUUID` in functions or a `identifiers` along with `UUID` as a type
- `isProduction` ?? `runtime` / `environment` or just stuff it into functions?